pinot-core/src/main/java/org/apache/pinot/core/query/scheduler/QueryScheduler.java

```diff
@@ -199,11 +199,13 @@ public void stop() {
 
     if (queryLogRateLimiter.tryAcquire() || forceLog(schedulerWaitMs, numDocsScanned)) {
       LOGGER.info(
-          "Processed requestId={},table={},segments(queried/processed/matched/consuming)={}/{}/{}/{},"
-              + "schedulerWaitMs={},totalExecMs={},totalTimeMs={},minConsumingFreshnessMs={},broker={},"
-              + "numDocsScanned={},scanInFilter={},scanPostFilter={},sched={}",
-          requestId, tableNameWithType, numSegmentsQueried, numSegmentsProcessed, numSegmentsMatched,
-          numSegmentsConsuming, schedulerWaitMs, timerContext.getPhaseDurationMs(ServerQueryPhase.QUERY_PROCESSING),
+          "Processed RequestId:{},table:{},segments(queried/processed/matched/consuming)={}/{}/{}/{},"
+              + "schedulerWaitMs:{},reqDeserMs:{},totalExecMs:{},resSerMs:{},totalTimeMs:{},minConsumingFreshnessMs:{},broker:{},"
```

Comment:
Can you add the new items at the end and not change the names of the variables? Is the changing of var names absolutely needed? 
If we are trying to minimize the line length, another way of doing this will be to create a separate logger that outputs in binary, and provide a utility to read it?

Reply:
@mcvsubbu, I didn't change any variable names. Have just added request deserialization (reqDeserMs) and response serialization (resSerMs) times. Nothing else has changed

Reply:
Also new items were not added at the end because I think it is better to keep the timing info together